docs/assistant/ai-esql-queries.asciidoc

```diff
@@ -0,0 +1,26 @@
+[[esql-queries-assistant]]
+= Generate, customize, and learn about {esql} queries with AI Assistant
+
+:frontmatter-description: Elastic AI Assistant can help you write ES|QL queries.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [guide]
+:frontmatter-tags-user-goals: [get-started]
+
+This page describes some of the ways Elastic AI Assistant can help you leverage the Elasticsearch Query Language ({esql}). 
+
+With AI Assistant's <<ai-assistant-knowledge-base, {esql} knowledge base>> enabled, AI Assistant benefits from specialized training data that enables it to answer questions related to {esql} at an expert level. 
+
+AI Assistant can help with {esql} in many ways, including:
+
+* **Education and training**: AI Assistant can serve as a powerful {esql} learning tool. Ask it for examples, explanations of complex queries, and best practices.
+* **Writing new queries**: Prompt AI Assistant to provide a query that accomplishes a particular task, and it will generate a query matching your description. For example: "Write a query to identify documents with `curl.exe` usage and calculate the sum of `destination.bytes`" or "What query would return all user logins to [a host] in the last six hours?"
```

Comment:
```suggestion
* **Writing new queries**: Prompt AI Assistant to provide a query that accomplishes a particular task, and it will generate a query that matches your description. For example: "Write a query to identify documents with `curl.exe` usage and calculate the sum of `destination.bytes`" or "What query would return all user logins to [a host] in the last six hours?"
```